code/controllers/subsystem/blackbox.dm

```diff
@@ -77,9 +77,9 @@ SUBSYSTEM_DEF(blackbox)
 
 /datum/controller/subsystem/blackbox/vv_edit_var(var_name, var_value)
 	switch(var_name)
-		if("feedback")
+		if(NAMEOF(src, feedback))
```

Comment:
Whats this even doing? Isn't var_name a string? NAMEOF will be the var rather than a string?

Reply:
This change comes from https://github.com/tgstation/tgstation/pull/52255. From my understanding, it gives the string like normal, but presumably errors if the var doesn't actually exist. I think this was done because the VV var checks were defined on the wrong types, being completely ineffective, whereas with this change those misplaced checks will clearly error preventing such issues from popping up in the first place.

Reply:
well it seems like how the other controllers do it, I have no idea how this works